src/definitions/location.ts

```diff
@@ -147,4 +148,14 @@ export type LocationDefinition = LocaleEntry<{
    * @see [IANA Time Zone Database](https://www.iana.org/time-zones)
    */
   time_zone: string[];
+
+  /**
+   * A list of languages.
```

Comment:
Should we use any language here or just the ones spoken in the locale?

```suggestion
   * A list of spoken languages.
```